icons/meson.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><title>file_type_meson</title><path d="M15.969,30v0C9.415,29.973,2,26.44,2,26.44,2.427,9.574,14.919,2.592,15.965,2.037V2L16,2.02,16.035,2v.04C17.081,2.6,29.573,9.578,30,26.443,30,26.443,22.537,30,15.969,30Z" style="fill:#39207c"/><path d="M17.8,23.1c.638.353.567,1.343.567,1.343-6.024.919-7.512-7.706-7.512-7.706a.9.9,0,0,1-1.346-.353c2.338-5.994,10.417-2.757,10.417-2.757-.443-.585.921-1.038.921-1.038C24.815,17.439,17.8,23.1,17.8,23.1Z" style="fill:#f7f7f7"/></svg>
```

Comment:
Please make sure that you pick a color from the [Material Design Color Palette](https://material.io/design/color/#tools-for-picking-colors). In addition, the inside of the icon should be transparent. I would suggest you using the following color `#3f51b5` to make the icon look like this:

On light background:

<img src="https://user-images.githubusercontent.com/12248527/78301030-c875c080-7538-11ea-80c2-c3f1843d019b.png" alt="meson file icon on light background" width="300px">

On dark background:

<img src="https://user-images.githubusercontent.com/12248527/78300878-85b3e880-7538-11ea-90f5-88e6d642a511.png" alt="meson file icon on dark background" width="300px">

You can find the transparent SVG files here:
https://github.com/mesonbuild/meson/blob/master/graphics/meson_logo.svg

Reply:
Applied material color and made inside transparent, please re-review